src/Build/Collections/WeakValueDictionary.cs

```diff
@@ -175,15 +174,13 @@ public bool TryGetValue(K key, out V value)
                 return true;
             }
 
-            value = wrappedValue.Target;
-
-            if (value == null)
+            if (!wrappedValue.TryGetTarget(out value))
             {
                 _dictionary.Remove(key);
                 return false;
             }
 
-            return result;
```

Comment:
Could get rid of `result` entirely, right?

```c#
             if (!_dictionary.TryGetValue(key, out WeakReference<V> wrappedValue))
             {
                 value = null;
                 return false;
             }
```

Reply:
Needed near the top of the method as a return from a TryGetValue()

Reply:
Oh wait duh I get it. Done in next push